stack.yaml

```diff
@@ -46,7 +46,11 @@ packages:
 
 # Dependency packages to be pulled from upstream that are not in the resolver
 # (e.g., acme-missiles-0.3)
-extra-deps: [bindings-gpgme-0.1.7]
+extra-deps:
+  - github: rethab/bindings-dsl
+    commit: 70c503ef67f0898c0f131e8ed83e334d688007a0
+    subdirs:
+      - bindings-gpgme
```

Comment:
after reading more about stack, I'm pretty sure this is not going to fly. Users of h-gpgme will only have our .cabal file, which doesn't know it needs to fetch this fork instead of the regular bindings-dsl. Therefore, those packages would fail to install.

Hopefully the PR in this repo gets merged. If not, we might have to follow one of the techniques described here: https://ro-che.info/articles/2015-02-09-dealing-with-broken-packages (releasing bindings-dsl under a new name bring the clear favourite as it means users can keep using h-gpgme without any extra steps).

Reply:
That is a good point. I think you are exactly right and I didn't think projects including h-gpgme.

At least this can be a reference of how to use the feature until the PR gets approved. https://github.com/jwiegley/bindings-dsl/pull/40

Reply:
This is now solved as I have taken over maintainership of bindings-dsl and just published a new version of bindings-gpgme.